integration_test/Cargo.toml

```diff
@@ -14,7 +14,8 @@ edition = "2021"
 download = ["node/download"]
 
 # Enable the same feature in `node` and the version feature here.
-# All minor releases of the latest three versions.
+# All minor releases of the latest four versions.
```

Comment:
Novel idea, increase this number so as not to have to make changes to the 4th oldest release. Are you planning on dropping this back to 3 or do you think we leave it as is for now?

Reply:
> Its a policy thing, if we keep this and 27.x comes out we have to add support for it. 

The comment below says `# Only the latest minor version for older versions.` which would suggest we have to anyway. I guess we don't want this for the older versions and should change the wording.